content/en/docs/concepts/scheduling/scheduling-framework.md

```diff
@@ -146,8 +143,11 @@ state, it will either trigger [Unreserve](#unreserve) plugins (on failure) or
 
 ### Permit
 
-These plugins are used to prevent or delay the binding of a Pod. A permit plugin
-can do one of three things.
+The Permit extension point contains two parts which spans scheduling cycle and
```

Comment:
Are developers able to customize both parts?

Reply:
not for the second(internal) WaitOnPermit phase. I use a dotted rectangle to represent that in the drawing.

BTW: I've followed @ahg-g's comment to restructure this paragraph.